pom.xml

```diff
@@ -58,6 +69,24 @@
             <version>${kotest.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.kotest.extensions</groupId>
+            <artifactId>kotest-assertions-arrow-jvm</artifactId>
```

Comment:
Kotest is suffering from the `-jvm` issue in combination with Maven? We should report this to Kotest.